website/src/_layouts/_base.njk

```diff
@@ -83,7 +83,8 @@
 											<a href="{{ item.link }}" class="mr-8 inline-flex items-center px-1 pt-1 border-b-2 border-transparent text-md font-medium leading-5 text-gray-500 hover:text-gray-700 hover:border-gray-300 focus:outline-none focus:text-gray-700 focus:border-gray-300 transition duration-150 ease-in-out">
 												<span>{{ item.title }}</span>
 
-												{% if item.pageType === 'support' %}															<span class="inline-block bg-indigo-500 rounded text-white ml-2 text-xs px-2 py-1">New!</span>
+												{% if item.pageType === 'support' %}
```

Comment:
This piece of code is dead, these lines can be deleted

Reply:
done: https://github.com/phpstan/phpstan/pull/11156/commits/8b390496083d090bc3911c35812c80724100a8a5